src/Controllers/ContentController.php

```diff
@@ -1537,16 +1561,25 @@ private function attachInstructorLessons(mixed $content, Request $request): mixe
         }
 
         $includedFields = [];
-        $includedFields[] = 'instructor,' . $content['id'];
+        $includedFields[] = 'instructor,'.$content['id'];
 
         $requiredFields = $request->get('required_fields', []);
         $includedFields = array_merge($request->get('included_fields', []), $includedFields);
         $requiredUserState = $request->get('required_user_states', []);
         $includedUserState = $request->get('included_user_states', []);
 
+        if($request->has('title')){
```

Comment:
Really minor point, but just a thought: would this possibly be better as `if($request->filled('title')){`? That way, we would avoid an additional condition of  `where title like "%%"` if the value is an empty string.